### DISCUSSION TITLE
I have a tab or blank space in a remote shell with oh my posh

### DISCUSSION THREAD

**Discussion by Santosgusta2010** [JanDeDobbeleer/oh-my-posh#5741] [General]
Hi, I have a Mac iTerminal and oh-my-posh on it. It works fine locally, but when I connect to the Mac using SSH the prompt is moved a few blank spaces to the right. Something like this:

![image](https://github.com/user-attachments/assets/bcbd0e1b-7ce0-4673-a54e-994ef8ecacee)

Locally it's working fine. Probably is something related to the config but I don't know where to start. 
Thanks in advance.

**Comment by JanDeDobbeleer** [comment]
@Santosgusta2010 probably something else is hijacking the prompt as well
